node/src/actors/chain_manager/mod.rs

```diff
@@ -262,6 +262,12 @@ pub struct ChainManager {
     initial_supply: u64,
     /// Populate RAD hashes index
     rad_hashes_index: bool,
+    /// Epoch when getSupplyInfo2 was last resolved.
```

Comment:
I suppose these variables are not saved to storage (and we do not need storage migrations)? That does imply whenever a node restarts (after a manual intervention or crash), this call will be slow again for the first time?